Userland/Libraries/LibJS/Bytecode/Pass/LoadElimination.cpp

```diff
@@ -29,8 +29,8 @@ static NonnullOwnPtr<BasicBlock> eliminate_loads(BasicBlock const& block, size_t
 
     auto new_block = BasicBlock::create(block.name(), block.size());
     HashMap<size_t, Register> identifier_table {};
-    HashMap<u32, Register> register_rerouting_table {};
-
+    static HashMap<u32, Register> register_rerouting_table {};
+    register_rerouting_table.clear();
```

Comment:
And now it's still cleared per block, so back on square one

Reply:
Im just confused where it should be cleared? After its definition is all the instructions. Should it be cleared on the call to `perform()`? Some insight would be very much appreciated. Thank you!

Reply:
Yes, it should be cleared on a call to `perform`, for that you will need to move the variable declaration out of the function